### ISSUE TITLE
Magisk 16.0 stock on Huawei nova 2s(HWI-AL00C01B192 (8.0.0.192)

### ISSUE THREAD

**Issue by Jackxu10000** [topjohnwu/Magisk#468]
I use twrp rec flash magisk-16.0(16.7)zip files and stock on Unlock Warning  screen,even without Huawei logo appear.And I try to patched RAMDISK.img which extractor from offical UPDATA.APP files ,you can download [from](http://update.hicloud.com:8180/TDS/data/files/p3/s15/G1915/g1789/v154009/f1/full/update.zip)
After I try,it seemed still not working and same as the first situation.
Which formal of info should I provided or How to get it.
Thanks for your help.

**Comment by topjohnwu** [comment]
Have you tried 16.7?

**Comment by Jackxu10000** [comment]
Yes,I have tried that but still not working,May be huawei's new GPU driver cause that,I have flashed my  phone to a lower version and everything work fine.

**Comment by heindrix** [comment]
I think it's happened because of new updates from Huawei (Patch01).
It detects root/Magisk.

**Comment by Jackxu10000** [comment]
Without Patch01 for 192 also cause the same problem.
